### PR TITLE
huggingface-cli 0.19.0

### PR DESCRIPTION
**Pull request by dnhn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/h/huggingface-cli.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class HuggingfaceCli < Formula
 
   desc "Client library for huggingface.co hub"
   homepage "https://huggingface.co/docs/huggingface_hub/index"
-  url "https://files.pythonhosted.org/packages/c8/6e/95d9518908ec9dcf7a97c0b0a14ddbf07e17158b3d4cefc522b91f823ae5/huggingface_hub-0.18.0.tar.gz"
-  sha256 "10eda12b9c1cfa800b4b7c096b3ace8843734c3f28d69d1c243743fb7d7a2e81"
+  url "https://files.pythonhosted.org/packages/d5/31/4ae775122014855aa626043bf348a5473fb80ce6492ca57017930eda98d5/huggingface_hub-0.19.0.tar.gz"
+  sha256 "7395c9b8053dc8aa0dec2900f367890f66be07bfe46369b95af3f4c1013eb09e"
   license "Apache-2.0"
   revision 1
```
Comment (chenrui333) [inline Formula/h/huggingface-cli.rb:9]:
```suggestion
```

File: Formula/h/huggingface-cli.rb
Comment (chenrui333) [inline Formula/h/huggingface-cli.rb:1]: resources also need updates.

**Review by chenrui333 — CHANGES_REQUESTED** [review]
`$ brew update-python-resources huggingface-cli`

**Comment by dnhn** [comment]
> `$ brew update-python-resources huggingface-cli`

This fails on my end. Please assist with this step.

**Comment by chenrui333** [comment]
> > `$ brew update-python-resources huggingface-cli`
> 
> This fails on my end. Please assist with this step.

can you post the logs?

**Review by chenrui333 — APPROVED** [review]

**Comment by dnhn** [comment]
> > > `$ brew update-python-resources huggingface-cli`
> > 
> > 
> > This fails on my end. Please assist with this step.
> 
> can you post the logs? (you might also need to have to run `brew update`)

Here is the log:
```
==> Retrieving PyPI dependencies for "huggingface-hub"...
Error: Unable to determine dependencies for "huggingface-hub" because of a failure when running
`/usr/local/Cellar/python@3.11/3.11.6_1/bin/python3 -m pip install -q --dry-run --ignore-installed --report=/dev/stdout huggingface-hub`.
Please update the resources manually.
```

**Pull request merged**